src/Microsoft.PowerShell.Commands.Utility/commands/utility/FormatAndOutput/common/GetFormatDataCommand.cs

```diff
@@ -169,27 +171,41 @@ protected override void ProcessRecord()
                 viewList.Add(formatdef);
             }// foreach(ViewDefinition...
 
-            // write out all the available type definitions
-            foreach (var pair in typedefs)
+            // If no Type Definitions are found for _typename throw a terminating error
+            if (typedefs.Count == 0)
             {
-                var typeNames = pair.Key;
-
-                if (writeOldWay)
+                // All files must have the same extension otherwise throw.
```

Comment:
Here the comment is unneeded. I believe we should remove it. (Or move to the method description).